index.js

```diff
@@ -2,6 +2,7 @@
 
 const inquirer = require('inquirer');
 const fs = require('fs');
+const v = require('voca');
```

Comment:
avoid single name variables

Reply:
Are you using voca anywhere in the file?

Reply:
If you need simple string manipulation try to avoid add dependencies to the mix.